packages/remark-mdx/test/fixtures/block-vs-inline.md

```diff
@@ -3,25 +3,22 @@
 To make this example more readable, uppercase letters are used when JSX is
 parsed as a block, wheres lowercase letters are used for inline JSX.
 
-This is inline, it’s a paragraph, because `</x>` is followed by `.`.
+This is inline, it’s a paragraph, because `<x>` is followed by `_`.
```

Comment:
I changed my mind here. I no longer think `<x># is this a heading?</x>` should be a heading. If you want blocks in tags, use blank lines.